p2p/proto/block.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+import "p2p/proto/state.proto";
+import "google/protobuf/timestamp.proto";
+
+// for now, we assume a small consensus, so this fits in 1M. Else, these will be repeated
+message Signatures {
+    BlockID block = 1;
+
+    repeated ConsensusSignature signatures = 2; //
+                                       // can be more explicit here about the signature structure as this is not part of account abstraction
+}
+
+// Note: commitments may change to be for the previous blocks like comet/tendermint
+// hash of block header sent to L1
+message BlockHeader {
```

Comment:
What about `gas_price`? I assume `starknet_version` is not necessary because if the protocol is updated "older" blocks will just use default values of the new fields?